convert.go

```diff
@@ -56,6 +57,27 @@ func FromNetAddr(a net.Addr) (ma.Multiaddr, error) {
 		// Encapsulate
 		return ipm.Encapsulate(udpm), nil
 
+	case "udt", "udt4", "udt6":
+		ac, ok := a.(*udt.UDTAddr)
+		if !ok {
+			return nil, errIncorrectNetAddr
+		}
+
+		// Get IP Addr
+		ipm, err := FromIP(ac.UDPAddr().IP)
```

Comment:
may want to abstract out this part common to `udp, utp, udt`. but i'm fine to merge